fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/boot/WebXmlConfiguration.java

```diff
@@ -46,10 +47,12 @@
     @Autowired
     private TenantAwareBasicAuthenticationFilter basicAuthenticationProcessingFilter;
 
+/*
     @Bean
     public Filter springSecurityFilterChain() {
         return new DelegatingFilterProxy();
     }
+*/
```

Comment:
Yep, I have such problem. I will do final self-review before commit more carefully.

Reply:
I saw that now it's cleanly removed instead of being commented out, but I still don't really understand why this `DelegatingFilterProxy` is not required anymore... particularly as this seems to be security related, I'm still curious to have a record of why?

Reply:
It is `DelegatingFilterProxy` duplicate. Spring security will create it by default.